modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc

```diff
@@ -40,10 +39,7 @@ To start using Dedicated, contact https://redpanda.com/try-redpanda?section=ente
 === Bring Your Own Cloud (BYOC)
 
 With BYOC clusters, you deploy Redpanda in your own VPC, so all data is
```

Comment:
This is only for BYO VPC. Re-word for standard BYOC.

Reply:
reworded: "With BYOC clusters, you deploy Redpanda in your own cloud (AWS or GCP),..."